packages/shared/lib/services/sync/run.service.ts

```diff
@@ -219,411 +212,274 @@ export default class SyncRun {
         optionalSecretKey?: string,
         optionalHost?: string
     ): Promise<ServiceResponse<boolean | object>> {
-        if (this.debug) {
-            const content = this.loadLocation ? `Looking for a local nango config at ${this.loadLocation}` : `Looking for a sync config for ${this.syncName}`;
-            if (this.writeToDb) {
-                await createActivityLogMessage({
-                    level: 'debug',
-                    environment_id: this.nangoConnection.environment_id,
-                    activity_log_id: this.activityLogId as number,
-                    timestamp: Date.now(),
-                    content
-                });
-                await this.logCtx?.debug(content);
-            } else {
-                logger.info(content);
-            }
-        }
-        let nangoConfig = this.loadLocation
-            ? await loadLocalNangoConfig(this.loadLocation)
-            : await getSyncConfig(this.nangoConnection, this.syncName, this.isAction);
```

Comment:
This is the main change, it is now done outside this method and simply ask syncConfig with or without loadLocation because we just need a few params not the full nango.yaml